### PR TITLE
Add card profile component

### PR DESCRIPTION
**Pull request by felipeorlando**
**CHANGELOG** :memo:

* Create `.card-profile` class component
* Update Card's documentation with Card Profile component

image

### CODE REVIEW

File: src/scss/components/_price_variation.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
     }
 
     &[class*='col-'] {
-      @include flex;
+      flex: 1;
```
Comment (nathpaiva) [inline src/scss/components/_price_variation.scss]: o `flex` da problema no IE, lá está ok?
qual é esse componente?

Comment (felipeorlando) [reply src/scss/components/_price_variation.scss]: Yes, this change comes from #153. In IE is working fine: 

image

File: src/scss/gaiden.scss [hunk 2]
```diff
@@ -37,6 +37,10 @@
 @import 'components/rating';
 @import 'components/card';
 @import 'components/card-citation';
+<<<<<<< HEAD
```
Comment (williamcosta) [inline src/scss/gaiden.scss]: 🔥

Comment (felipeorlando) [reply src/scss/gaiden.scss]: 💧

**Pull request merged**